### ISSUE TITLE
Convex hull with repeated -0 entry is wrong

### ISSUE THREAD

**Issue by mforets** [JuliaReach/LazySets.jl#892]
```julia
julia> convex_hull([[0.0, -1.0],
             [-0.0, 1.0],
             [0.0, 1.0]])

2-element Array{Array{Float64,1},1}:
 [-0.0, 1.0]
 [0.0, 1.0] 
```

**Comment by schillic** [comment]
Wrong in which sense? That it contains duplicates?

**Comment by mforets** [comment]
This is now fixed:

```julia
julia> convex_hull([[0.0, -1.0],
                    [-0.0, 1.0],
                    [0.0, 1.0]])
2-element Array{Array{Float64,1},1}:
 [-0.0, 1.0]
 [0.0, -1.0]
```

**Comment by schillic** [comment]
No, only the three-vertex case is fixed (simply because we use a different algorithm). The old code was not touched.
```julia
julia> convex_hull([[0.0, -1.0],
                    [-0.0, 1.0],
                    [0.0, 1.0],
                    [0.0, -1.0]])
2-element Array{Array{Float64,1},1}:
 [-0.0, 1.0]
 [0.0, 1.0]
```

**Comment by schillic** [comment]
Seems to be fixed.
```julia
julia> convex_hull([[0.0, -1.0],
                    [-0.0, 1.0],
                    [0.0, 1.0],
                    [0.0, -1.0]])
2-element Array{Array{Float64,1},1}:
 [0.0, -1.0]
 [-0.0, 1.0]
```
Not sure who fixed it. I'll assign all of us :smile:

**Comment by schillic** [comment]
Argh, that was again a case that got resolved by adding a specific algorithm, this time for four vertices. Here is a new example:
```julia
julia> convex_hull([[0.0, -1.0], [-0.0, 1.0], [0.0, 1.0], [0.0, -1.0], [0.0, 0.0]])
2-element Array{Array{Float64,1},1}:
 [-0.0, 1.0]
 [0.0, 1.0] 
```

**Comment by mforets** [comment]
In `master`, i now get:

```julia
julia> convex_hull([[0.0, -1.0], [-0.0, 1.0], [0.0, 1.0], [0.0, -1.0], [0.0, 0.0]])
1-element Array{Array{Float64,1},1}:
 [-0.0, 1.0]
```

**Comment by schillic** [comment]
Seems to have been fixed in #1718.
